docs/receiving/receiving-with-ingest.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: Using the Ingest API
+---
+
+import CodeTabs from '@theme/CodeTabs';
+import TabItem from '@theme/TabItem';
+
+[Svix Ingest] grants you scalability, security best practices, and
+reliability, all with minimal configuration.
+
+Creating a `Source` will generate an endpoint you can share with a webhook
+provider as a destination for their webhooks.
+
+Ingest supports a variety of signature verification schemes used by webhook
+providers. Supported values for a `Source`'s' `type` include:
+
+- `beehiiv`
+- `brex`
+- `clerk`
+- `github`
+- `guesty`
+- `hubspot`
+- `incidentIo`
+- `lithic`
+- `nash`
+- `pleo`
+- `replicate`
+- `resend`
+- `safebase`
+- `sardine`
+- `shopify`
+- `stripe`
+- `stych`
+- `svix` (that's us!)
+- `zoom`
+
+Additionally there's the option to configure a `Source` as `http` that doesn't
+perform any signature verification at all for cases where the provider has no
```

Comment:
```suggestion
perform any signature verification. This is useful for providers that have no verification scheme
and for providers whose verification scheme is not yet supported by 
Svix Ingest (please contact us so we can add it!).
```

Reply:
N.b. I have a callout section `:::tip` immediately after this paragraph to ask for new schemes.

You want I should remove the `:::tip`? I'm more inclined to leave it and snip the last line out of your suggestion.

Here's the current text rendered out:
![image](https://github.com/user-attachments/assets/28bec9e2-0925-4be6-9577-4e9e0a63de01)